### DISCUSSION TITLE
启动后无回应,只会回应emojimix插件

### DISCUSSION THREAD

**Discussion by enzitdr456** [nonebot/discussions#418] [问答]
![image](https://github.com/nonebot/discussions/assets/133577149/167d98b2-6989-4f64-9d3a-b4ffa59ffe5b)
测试了很多插件都没有回应,唯独emojimix插件会回应,目前新项目只安装了三个插件:
[tool.nonebot]
adapters = [
    { name = "OneBot V11", module_name = "nonebot.adapters.onebot.v11" }
]
plugins = ["nonebot_plugin_picstatus", "nonebot_plugin_emojimix"]
plugin_dirs = []
builtin_plugins = ["echo"]

**Comment by enzitdr456** [comment]
![image](https://github.com/nonebot/discussions/assets/133577149/d111c159-9edf-4fb8-a5cf-c390a94f3a60)
补充信息
.env.prod内容:
DRIVER=~fastapi
PORT=8013

**Reply by yanyongyu** [reply]
请检查你的配置，如：`COMMAND_START`, `SUPERUSERS`

**Reply by imvanda** [reply]
同样的问题，我只有echo插件工作，请问解决了吗？

**Reply by KlNon** [reply]
我也是,不过我的只有echo工作,我的配置文件如下:
```
HOST=0.0.0.0
PORT=5777
# LOG_LEVEL=DEBUG
FASTAPI_RELOAD=false

SUPERUSERS=["我的QQ号"]  # 配置 NoneBot 超级用户
NICKNAME=["bot"]  # 配置机器人的昵称
COMMAND_START=["/", "."]
COMMAND_SEP=[".", " "]

driver=~httpx+~websockets

# Mirai2
ENVIRONMENT=dev
VERIFY_KEY=1234567890  # MiraiApiHttp2 配置文件里的 token

MIRAI_HOST=0.0.0.0  # MiraiApiHttp2 的 ip
MIRAI_PORT=5700  # MiraiApiHttp2 的端口
MIRAI_QQ=["机器人QQ号"]  # Mirai 上已经登录的 qq 号
```
![image](https://github.com/nonebot/discussions/assets/58141387/3258f249-446a-4632-aaa6-8d757d4e1712)

**Reply by KlNon** [reply]
![image](https://github.com/nonebot/discussions/assets/58141387/56cb90ab-63f2-4446-8fb0-03748d77c1ef)
我输入命令的后台结果是这样的,没有回复我,除了使用echo
![image](https://github.com/nonebot/discussions/assets/58141387/9aa527b6-96bb-49fa-acc7-7ff109a0dfe5)
![image](https://github.com/nonebot/discussions/assets/58141387/fcd86ec5-1118-47ed-9158-34b7862d353e)

**Reply by yanyongyu** [reply]
你可以打开debug日志，我猜是插件不支持mirai2适配器
